### PR TITLE
Kankuro/dev

### PR DESCRIPTION
**Pull request by safaci2000**

### CODE REVIEW

File: commands/poll-commands/addpoll.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// // Import the required files
```
Comment (safaci2000) [inline commands/poll-commands/addpoll.js:1]: Remove this

File: commands/poll-commands/listpolls.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// // Import the required files
```
Comment (safaci2000) [inline commands/poll-commands/listpolls.js:1]: Remove section or file as appropriate.

File: commands/poll-commands/removepoll.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+// // Import the required files
```
Comment (safaci2000) [inline commands/poll-commands/removepoll.js:1]: same

File: controllers/AutorolesController.js [hunk 4]
```diff
@@ -1,19 +1,15 @@
...
     // Create a function with required args
-    autoroleHandler: function(cmd, s, c, a, m) {
+    autoroleHandler: function(cmd, c, a, m) {
         // Create vars
-        const command = cmd;
-        const sequelize = s;
-        const client = c;
-        const args = a;
-        const message = m;
+        const command = cmd, client = c, args = a, message = m;
```
Comment (safaci2000) [inline controllers/AutorolesController.js:12]: I guess you can't pass the function arguments as a const. 

You may want to look at: https://immutable-js.github.io/immutable-js/

It'd be nice to be able to just pass an object and simply use the object.

req.command = 
req.message = ..

etc.  Future improvement I suppose

File: controllers/AutorolesController.js [hunk 5]
```diff
@@ -24,29 +20,11 @@ module.exports = {
...
-        {
-            charset: 'utf8mb4',
-            collate: 'utf8mb4_bin',
-        });
 
         /*********** ADD AUTOROLE ***********/
         if (command.name === 'addautorole') {
             // Search for the role within the server
-            const role = message.guild.roles.find(role => role.name.toLowerCase() === autorole);
+            const role = message.guild.roles.cache.find(role => role.name.toLowerCase() === autorole);
```
Comment (safaci2000) [inline controllers/AutorolesController.js:27]: what's the behavior of the cache? if a Role is removed from discord, how'd you handle that or essentially when 

are you updating/clearing the cache?

File: controllers/DatabaseController.js [hunk 6]
```diff
@@ -1,21 +1,25 @@
 // Import the required files
 const Sequelize = require('sequelize');
-const {prefix, db_name, db_host, db_port, db_user, db_pass} = require("../config.json");
+const moment = require("moment");
+const {prefix, db_name, db_host, db_port, db_user, db_pass, action_log_channel} = require("../config");
 const TriggersController = require("./TriggersController");
 const AutorolesController = require("./AutorolesController");
 const JoinableRolesController = require("./JoinableRolesController");
 const WarningsController = require("./WarningsController");
+//const PollsController = require("./PollsController");
```
Comment (safaci2000) [inline controllers/DatabaseController.js:9]: remove

File: controllers/DatabaseController.js [hunk 7]
```diff
@@ -39,7 +43,16 @@ module.exports = {
...
 
             // Call the trigger handler function from the TriggersController file
-            TriggersController.triggerHandler(command, sequelize, client, args, message, triggerList);
+            TriggersController.triggerHandler(command, client, args, message, triggerList);
+
+        /*
+        ###################################
+        ########## poll commands ##########
+        ###################################
+        */
```
Comment (safaci2000) [inline controllers/DatabaseController.js:52]: remove

File: controllers/PollsController.js [hunk 8]
```diff
@@ -0,0 +1,152 @@
+// // Import the required files
```
Comment (safaci2000) [inline controllers/PollsController.js:1]: Remove file

File: controllers/AutorolesController.js (hunk 5 above)
Comment (TobyHorn) [reply controllers/AutorolesController.js:27]: Tbh, I'm not 100% sure, it is a new thing they introduced with v12.

https://discordjs.guide/additional-info/changes-in-v12.html#managers-cache

File: controllers/AutorolesController.js (hunk 4 above)
Comment (TobyHorn) [reply controllers/AutorolesController.js:12]: You don't have to assign the passed in param to a local var, this is just a bad habit of mine tbh.

**Comment by TobyHorn** [comment]
I'll handle the cleanup mentioned in previous comments once my last commit is fixed (reverted to what you had before I pushed last)

Not sure if I'm just an idiot or if vscode messed up (or both), but yea, sorry about that. Going forward I'd prefer to just work on my own branch that others don't touch to avoid conflict such as this.

**Pull request merged**